apps/condo/domains/meter/utils/taskSchema/DomaMetersImporter.js

```diff
@@ -64,6 +69,7 @@ class DomaMetersImporter extends AbstractMetersImporter {
                 commissioningDate: row[15],
                 sealingDate: row[16],
                 controlReadingsDate: row[17],
+                isAutomatic: isEmpty(cell19Value) ? undefined : this.mappers.isAutomatic[cell19Value],
```

Comment:
Incorrect comparison.

In line 40:
```
const cell19Value = String(null).toLowerCase() // "null"
```

```
isEmpty("null") // false -> this.mappers.isAutomatic['null']
```

Reply:
this.mappers.isAutomatic['null'] -> undefined -> ignored of backend as empty value